File: inst/build_module/4_moduleServer/app_hist.R
library(geyser)

app_ui <- shiny::bootstrapPage(
  histInput(id = "hist"), 
  histOutput(id = "hist"),
  histUI(id = "hist")
)
app_server <- function(input, output, session) {
  histServer(id = "hist")
}
shiny::shinyApp(app_ui, app_server)

File: inst/build_module/5_python/app_hist.py
from shiny import App, render, ui
from geyser.hist import hist_server, hist_input, hist_output, hist_ui
import geyser.io as io

app_ui = ui.page_fluid(
    hist_input("hist"),
    hist_output("hist"),
    hist_ui("hist")
)

def app_server(input, output, session):
    hist_server("hist")

app = App(app_ui, app_server)

io.app_run(app)
